controllers/search.js

```diff
@@ -0,0 +1,51 @@
+const searchService = require('../services/search');
+const statusCode = require('../constants/statusCode');
+const logger = require('../modules/logger');
+
+const env = process.env.NODE_ENV || 'development';
+const config = require('../config/config.js')[env];
+
+const { swapValues } = require('./utility');
+
+module.exports = {
+
+  searchSellers: (params) => {
+    return new Promise((resolve, reject) => {
+      const { latitude, longitude, searchText } = params;
+      if (!latitude || !longitude || !searchText) {
+        return reject(['Missing params', statusCode.BAD_REQUEST]);
+      }
+
+      /**
+       * A threshold = 0.3 gives a distance range ~ 47 KM
+       * since latitude and longitude can be negative so check 
+       * which one is greater after applying threshold
+       */
+      const latitudeMax = parseFloat(latitude) + config.latitudeThreshold;
+      const latitudeMin = parseFloat(latitude) - config.latitudeThreshold;
+      let latitudeRange = [latitudeMin, latitudeMax];
+      if (latitudeMin > latitudeMax) {
+        // swap values
```

Comment:
We should move this latitude range calculation in a helper function, let's keep it for another PR.

Reply:
sure